compose.yaml

```diff
@@ -31,7 +31,7 @@ services:
       - ILIOS_DATABASE_URL=mysql://ilios:ilios@db/ilios
       - ILIOS_REQUIRE_SECURE_CONNECTION=false
       - ILIOS_ERROR_CAPTURE_ENABLED=false
-      - ILIOS_SEARCH_HOST=http://opensearch:9200
+      - ILIOS_SEARCH_HOSTS=http://opensearch:9200
```

Comment:
Singular is correct here, this go [renamed](https://github.com/ilios/ilios/blob/master/docs/update.md#upgrading-to-ilios-31230) in v3.123.0.